ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```diff
@@ -15,7 +15,7 @@ base_images:
     name: centos
```

Comment:
Do we still need it?

Reply:
Instead of adding I can just replace the os with the new one to avoid any confusion